client/src/components/Signup/Signup.js

```diff
@@ -70,7 +70,7 @@ class Signup extends Component {
                         <Input type="text" name="classCode" id="classCode" placeholder="Class Code" value={this.props.classCode} onChange={this.props.handleInputChange} valid={this.state.validClassCode} />
                     </FormGroup>
                     <FormGroup>
-                        <Label for="fName">Last Name</Label>
```

Comment:
fName means first name. There needs to be a separate label for last name and first name